packages/app-store/routing-forms/lib/jsonLogicOverrides.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import jsonLogic from "json-logic-js";
+
+// converts input to lowercase if string
+function normalize<T extends string | string[]>(input: T): T {
+  if (typeof input === "string") {
+    return input.toLowerCase() as T;
+  }
+  if (input instanceof Array) {
```

Comment:
A failing test caught this bug where input can sometimes be an array as well(for MultiSelect it is)

Reply:
Got it.